extensions/wikia/PortableInfoboxBuilder/js/PortableInfoboxBuilderPonto.js

```diff
@@ -9,24 +9,23 @@ define('wikia.infoboxBuilder.ponto', ['wikia.window', 'ponto'], function (window
 		this.isWikiaContext = function () {
 			return {
 				isWikiaContext: true,
-				isLoggedIn: window.wqUserName !== null ? true : false
-			}
+				isLoggedIn: w.wqUserName !== null
+			};
 		};
 
 		/**
 		 * redirects to template page
-		 * @param {String} title
 		 * @returns {Boolean}
 		 */
-		this.redirectToTemplatePage = function (title) {
-			window.location = window.location.origin + '/wiki/Template:' + title;
+		this.redirectToTemplatePage = function () {
+			w.location = w.templatePageUrl;
```

Comment:
Is it possible that this won't be available? Should we check it first to be sure we don't redirect to undefined?